messagingpush/src/main/java/io/customer/messagingpush/processor/PushMessageProcessorImpl.kt

```diff
@@ -89,7 +88,7 @@ internal class PushMessageProcessorImpl(
         if (moduleConfig.autoTrackPushEvents) {
             eventBus.publish(
                 Event.TrackPushMetricEvent(
-                    event = MetricEvent.delivered.name,
+                    event = Metric.Delivered.name,
```

Comment:
@Shahroz16 Do we need to use `Metric.Delivered.name.lowercase(Locale.ENGLISH)` instead for this and similar changes? Or will it work without the transformation?

Reply:
I think we need a class that lives in `core` with the format that CDP expects, so then we can just use name easily everywhere

Reply:
Added extension method